docs/commands/json.merge.md

```diff
@@ -1 +1,115 @@
-Sets or updates the JSON value at a path.
+Merge one or more `value` in a JSON document adding, updating, deleting and/or replacing a array at `path` in `key`
+
+Compliance with Json Merge Patch [RFC7396](https://datatracker.ietf.org/doc/html/rfc7396)
```

Comment:
This command complies with [RFC 7396 - JSON Merge Patch](https://datatracker.ietf.org/doc/html/rfc7396).